pureftpd/symlink check/check_symlink_accounts.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+MYSQL=/usr/bin/mysql
+LOGFILE=/var/log/symlink.log
+TS=`date '+%Y-%m-%d %H:%M:%S'`
+sylinks=0
+$MYSQL --defaults-file=/etc/mysql/debian.cnf -N -e "use pureftpd; select Dir from users;" | \
```

Comment:
This will not work on RHEL based systems. You don't need the `--defaults-file=/etc/mysql/debian.cnf` specified. By default on our systems you can login without password if the script is running as `root`. This is not that secure but is the way that we are using it at the moment.